src/style/pauseable_placement.js

```diff
@@ -63,7 +63,10 @@ class PauseablePlacement {
         while (this._currentPlacementIndex >= 0) {
             const layerId = order[this._currentPlacementIndex];
             const layer = layers[layerId];
-            if (layer.type === 'symbol') {
+            const placementZoom = this.placement.collisionIndex.transform.zoom;
+            if (layer.type === 'symbol' &&
+                (!layer.minzoom || layer.minzoom <= placementZoom) &&
+                (!layer.maxzoom || layer.maxzoom > placementZoom)) {
```

Comment:
a small thing: when `layer.maxzoom === 0` wouldn't be handled correctly because `!layer.maxzoom` would`true`. This seems like a super unlikely but technically valid value.

Reply:
Good point -- but `style_layer.js` and `worker_tile.js` both have the same behavior (treating `maxzoom === 0` as if there's no maxzoom set), so I think I'll err on the side of consistency for now.